sycl/test/scheduler/HostAccDestruction.cpp

```diff
@@ -0,0 +1,38 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: env SYCL_PI_TRACE=1 %CPU_RUN_PLACEHOLDER %t.out 2>&1 %CPU_CHECK_PLACEHOLDER
```

Comment:
I think running on cpu device should be enough. Suggest removing other run lines.